pkgs/development/libraries/at-spi2-atk/default.nix

```diff
@@ -1,24 +1,24 @@
-{ stdenv, fetchurl, python, pkgconfig, popt, atk, libX11, libICE, xorg, libXi
-, intltool, dbus-glib, at-spi2-core, libSM }:
+{ stdenv, fetchurl, meson, ninja, pkgconfig
+, at-spi2-core, atk, dbus, glib, libxml2 }:
 
 stdenv.mkDerivation rec {
-  versionMajor = "2.26";
-  versionMinor = "1";
-  moduleName   = "at-spi2-atk";
   name = "${moduleName}-${versionMajor}.${versionMinor}";
+  moduleName   = "at-spi2-atk";
+  versionMajor = "2.26";
+  versionMinor = "2";
 
   src = fetchurl {
     url = "mirror://gnome/sources/${moduleName}/${versionMajor}/${name}.tar.xz";
```

Comment:
It is better to use `stdenv.lib.versions.majorMinor version` so that the various update tools could update the single `version` attribute, see https://github.com/NixOS/nixpkgs/issues/36150.